Actors/VisualObjects/VisualObjects/ApplicationParameters/Cloud.xml

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Application xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="fabric:/VisualObjects" xmlns="http://schemas.microsoft.com/2011/01/fabric">
+   <Parameters>
+      <Parameter Name="VisualObjects.WebService_InstanceCount" Value="-1" />
+      <Parameter Name="VisualObjects.ActorService_PartitionCount" Value="10" />
```

Comment:
If the default is 10 and 10 then each partition only has 1 object, and also since the default cluster size is 5 then 2 partitions get upgraded simultaneously. Consider making this 5 or also upping the object count for the cloud profile.